staging/src/k8s.io/apiserver/pkg/audit/policy/reader.go

```diff
@@ -55,17 +55,21 @@ func LoadPolicyFromFile(filePath string) (*auditinternal.Policy, error) {
 		return nil, fmt.Errorf("failed to read file path %q: %+v", filePath, err)
 	}
 
+	return LoadPolicyFromBytes(policyDef, filePath)
+}
+
+func LoadPolicyFromBytes(policyDef []byte, location string) (*auditinternal.Policy, error) {
```

Comment:
location doesn't make much sense in this function signature. The caller should append the location to the error message IMO.

Reply:
> location doesn't make much sense in this function signature. The caller should append the location to the error message IMO.

Wrapping errors in golang loses the type of the original error for error type checks.  I could wrap it like that here, but generally that's a really bad thing for a library to do.